qm-plugins/qm-object-cache/html/class-qm-output-object-cache-ops.php

```diff
@@ -0,0 +1,156 @@
+<?php
+/**
+ * Output class
+ *
+ * Class QM_Output_Operations_ObjectCache_Ops
+ */
+class QM_Output_ObjectCache_Ops extends QM_Output_Html {
+
+	public function __construct( QM_Collector $collector ) {
+		global $wp_object_cache;
+		if ( ! method_exists( $wp_object_cache, 'get_stats' ) ) {
+			return;
+		}
+
+		parent::__construct( $collector );
+
+		add_filter( 'qm/output/menu_class', array( $this, 'admin_class' ) );
+		add_filter( 'qm/output/panel_menus', array( $this, 'panel_menu' ), 99 );
+	}
+
+	/**
+	 * Outputs data in the footer
+	 */
+	public function output() {
+		$data = $this->collector->get_data();
+		if ( empty( $data ) ) {
+			return;
+		}
+
+		$ops    = $data['operations'] ?? [];
+		$groups = $data['groups'] ?? [];
+
+		$this->before_tabular_output();
+
+		echo '<thead>';
+		echo '<tr>';
+		$this->output_filterable_table_col( 'Operation', array_keys( $ops ) );
+		$this->output_sortable_table_col( 'Key' );
+		$this->output_sortable_table_col( 'Size' );
+		$this->output_sortable_table_col( 'Time' );
+		$this->output_filterable_table_col( 'Group', array_keys( $groups ) );
+		$this->output_sortable_table_col( 'Result' );
```

Comment:
Should these column header strings be made translatable the same as some of the menu/title strings?